tests/src/integration/runtimetests/manifest.yaml

```diff
@@ -158,6 +162,9 @@ packages:
             helloworldjava-without-explicit-runtime:
                 function: src/hello.jar
                 main: Hello
+            helloworlddotnet22-without-explicit-runtime:
```

Comment:
I think we don't need this entry, you can remove it.
For dotnet is always explicit because is only .zip, there is no way to deduce the runtime using .zip